utl/blogs.py

```diff
@@ -3,22 +3,42 @@
 
 __dbfile__ = os.path.dirname(os.path.abspath(__file__)) + '/../data/sitedata.db'
 
+def init():
+    db = sqlite3.connect(__dbfile__)
+    db.execute('CREATE TABLE IF NOT EXISTS blogs (userid INTEGER, blogid PRIMARY KEY, title TEXT UNIQUE);')
+    db.commit()

```

Comment:
is the execute command the same one that you used when you manually created the table?